amazon-kinesis-client-multilang/src/test/java/software/amazon/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -34,12 +31,25 @@
 import junit.framework.Assert;
 import software.amazon.awssdk.auth.credentials.AwsCredentials;
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.services.kinesis.model.InvalidArgumentException;
 import software.amazon.kinesis.multilang.config.KinesisClientLibConfigurator;
-import software.amazon.kinesis.multilang.config.MultiLangDaemonConfiguration;
 
 @RunWith(MockitoJUnitRunner.class)
 public class MultiLangDaemonConfigTest {
     private static String FILENAME = "some.properties";
+    private static String TestExe = "TestExe.exe";
+    private static String TestApplicationName = "TestApp";
+    private static String TestStreamName = "fakeStream";
+    private static String TestStreamNameInArn = "FAKE_STREAM_NAME";
+    private static String TestRegion = "us-east-1";
+    private static String TestRegionInArn = "us-east-2";
+
+    private static String getTestStreamArn(){
+        return String.format("arn:aws:kinesis:%s:ACCOUNT_ID:stream/%s", TestRegionInArn, TestStreamNameInArn);
```

Comment:
Why have this static method that always computes the same value? KISS with a member variable.